src/components/four-up.tsx

```diff
@@ -238,15 +238,17 @@ export class FourUpComponent extends BaseComponent<IProps, IState> {
 
     const memberName = (groupUser?: GroupUserModelType) => {
       const userFocused = isFocused(groupUser);
-      if (userFocused || !groupUser) {
+      if ((userFocused && viaStudentGroupView) || !groupUser) {
         return null;
       }
 
       const { name: fullName, initials } = groupUser;
+      const className = classNames("member", {"member-centered": userFocused && !viaStudentGroupView});
```

Comment:
This was a bug introduced in the switch of the 4-up button - we still need to show the student name overlayed when we are in the teacher dashboard.